foobar

```diff
@@ -1,14 +1,12 @@
 Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod
-tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At
-vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren,
-no sea takimata sanctus est Lorem ipsum dolor sit amet.
-
 Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod
-tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At
-vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren,
-no sea takimata sanctus est Lorem ipsum dolor sit amet.
-
 Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod
+no sea takimata sanctus est Lorem ipsum dolor sit amet.
+no sea takimata sanctus est Lorem ipsum dolor sit amet.
+no sea takimata sanctus est Lorem ipsum dolor sit amet.
+tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At
+tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At
```

Comment:
Unclear what this change brings to the project. Can you explain in more detail?

Reply:
it makes the codes more readables

Reply:
me no agree

Reply:
mmm